app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -92,27 +84,23 @@ class ConversationListViewModel @Inject constructor(
     }
 
     private fun startObservingConversationsAndConnections() = viewModelScope.launch {
-        observeConversationsAndConnections()
-            .combine(observeSelfUser(), ::Pair)
+        observeConversationViewUseCase()
             .flowOn(dispatcher.io())
-            .collect { (conversationListDetails, selfUser) ->
-                with(conversationListDetails) {
-                    selfUserId = selfUser.id
-                    state = ConversationListState(
-                        conversations = conversationList.toConversationsFoldersMap(selfUser),
-                        missedCalls = mockMissedCalls, // TODO: needs to be implemented
-                        callHistory = mockCallHistory, // TODO: needs to be implemented
-                        unreadMentions = mockUnreadMentionList, // TODO: needs to be implemented
-                        allMentions = mockAllMentionList, // TODO: needs to be implemented
-                        newActivityCount = unreadConversationsCount,
-                        unreadMentionsCount = mentionsCount, // TODO: needs to be implemented on Kalium side
-                        missedCallsCount = missedCallsCount // TODO: needs to be implemented on Kalium side
-                    )
-                }
+            .collect { conversationListDetails ->
+                state = ConversationListState(
+                    conversations = conversationListDetails.toConversationsFoldersMap(),
+                    callHistory = mockCallHistory, // TODO: needs to be implemented
+                    unreadMentions = mockUnreadMentionList, // TODO: needs to be implemented
+                    allMentions = mockAllMentionList, // TODO: needs to be implemented
+                    newActivityCount = 0L,
+                    unreadMentionsCount = 0L, // TODO: needs to be implemented on Kalium side
```

Comment:
Wasn't this also implemented?

Reply:
it was a todo: // TODO: needs to be implemented on Kalium side

Reply:
This was implemented a while ago by me, we were able to get newActivityCount which was unreadConversationsCount, because at the moment I implemented we only had support for that